### PR TITLE
Fix a couple of typos found in the documentation

### PR DESCRIPTION
**Pull request by EPadronU**
[Review on Reviewable](https://reviewable.io/reviews/ratpack/ratpack/855)

### CODE REVIEW

**Comment by danhyun** [comment]
Thanks, really appreciate it!

For ratpack-manual/src/content/chapters/30-guice.md:19

I would leave as

> means of decoupling

Or change to

> way to decouple

Can you please add yourself to the contributors list.

File: ratpack-manual/src/content/chapters/30-guice.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The `ratpack-guice` library provides the [`BindingsSpec`](api/ratpack/guice/Bind
 
 ## Dependency injected handlers
 
-The Guice integration gives you a means of decoupling the components of your application.
+The Guice integration gives you a mean to decouple the components of your application.
```
Comment (danhyun) [inline ratpack-manual/src/content/chapters/30-guice.md]: I would leave as "means of decoupling" or change to "way to decouple"

**Comment by EPadronU** [comment]
Hi, @danhyun. Should I modify this PR in order to introduce "way to decouple" as you have mentioned, and to include myself in the contributors list?

**Comment by danhyun** [comment]
Yes please :)

**Comment by EPadronU** [comment]
Done. @danhyun

**Comment by danhyun** [comment]
Thanks again.

**Comment by EPadronU** [comment]
My pleasure.

**Pull request merged**